gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
It’s settled then. Complete rewrite of our gulp pipeline. Me like. Side-remark: Gulp 4 also brings native support for task *sequences*.